code/game/mecha/equipment/tools/medical_tools.dm

```diff
@@ -114,6 +114,12 @@
 		return "[output] [temp]"
 	return
 
+/obj/item/mecha_parts/mecha_equipment/tool/sleeper/alt_action()
+	if(!occupant)
```

Comment:
Could be moved directly to the spell's `cast_check`, but that's not a major concern.